examples/nftvm/storage/marketplace_storage.go

```diff
@@ -0,0 +1,94 @@
+// Contains read/write logic pertaining to the NFT marketplace
+package storage
+
+import (
+	"context"
+	"encoding/binary"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/hashing"
+	"github.com/ava-labs/hypersdk/codec"
+	"github.com/ava-labs/hypersdk/consts"
+	"github.com/ava-labs/hypersdk/state"
+)
+
+const MarketplaceOrderStateChunks = consts.Uint64Len
+
+// TODO: tune this
+const BuyNFTStateChunks = 50
+
+func MarketplaceOrderStateKey(
+	orderID ids.ID,
+) (k []byte) {
+	k = make([]byte, consts.Uint16Len+ids.IDLen+consts.Uint16Len)
+	k[0] = marketplaceOrderPrefix
+	copy(k[1:], orderID[:])
+	binary.BigEndian.PutUint16(k[1+ids.IDLen:], MarketplaceOrderStateChunks)
+	return
+}
```

Comment:
```suggestion
) ([]byte) {
	k := make([]byte, consts.Uint16Len+ids.IDLen+consts.Uint16Len)
	k[0] = marketplaceOrderPrefix
	copy(k[1:], orderID[:])
	binary.BigEndian.PutUint16(k[1+ids.IDLen:], MarketplaceOrderStateChunks)
	return k
}
```

same comment throughout - we more commonly use this style rather than naming the return parameter. This makes it easier to follow imo